js/core/src/tracing.ts

```diff
@@ -42,12 +42,41 @@ const instrumentationKey = '__GENKIT_TELEMETRY_INSTRUMENTED';
  * @hidden
  */
 export async function ensureBasicTelemetryInstrumentation() {
+  await checkFirebaseMonitoringAutoInit();
```

Comment:
There is some very relevant refactoring happening: https://github.com/firebase/genkit/pull/3062
But I think this change is probably OK. If we go ahead with that refactor, we'll just move this call into node implementation of instrumentation.

Reply:
Let me know if I should wait for that, or if this can go.